sdk/eventhubs/.dict.txt

```diff
@@ -2,3 +2,6 @@ amqps
 consts
 eventdata
 mgmt
+amqp
```

Comment:
Yeah - just put "amqp" in the general location for all words. It's general enough within Azure SDKs. I could see keeping "amqps" isolated for now, though.